src/rpc/handlers/AccountNFTs.cpp

```diff
@@ -79,7 +79,7 @@ AccountNFTsHandler::process(AccountNFTsHandler::Input input, Context const& ctx)
     auto const blob = sharedPtrBackend_->fetchLedgerObject(pageKey, lgrInfo.seq, ctx.yield);
 
     if (!blob)
```

Comment:
If the blob is empty or does not exist , maybe because this account does not have any NFT. In this case, the marker is not given, we should not return error. 
If the blob is not empty ,maybe its type is not NFT page. In this case, we should return error. So I think we need to check the type somewhere.